bin/install_travis.sh

```diff
@@ -90,5 +93,5 @@ if [[ "${WITH_PIRANHA}" == "yes" ]]; then
     git clone https://github.com/bluescarni/piranha;
     cd piranha && mkdir build && cd build;
     cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$our_install_dir -DBUILD_TESTS=no ../ && make -j8 install && cd ../..;
-fi
+fi	
```

Comment:
In #678  I ignored `.sh` files of the `bin` folder . Should I remove that exception and check for `.sh` files too? That would have prevented this error.